package.json

```diff
@@ -36,6 +36,7 @@
     "chai": "^3.2.0",
     "chai-as-promised": "^5.0.0",
     "eslint": "^0.24.1",
+    "eslint-plugin-react": "^3.9.0",
```

Comment:
Is this really needed, or was there something odd with your local eslint installation? There's nothing related to React in this project.